src/index.js

```diff
@@ -48,7 +48,8 @@ class ReactIntlUniversal {
       warningHandler: console.warn, // ability to accumulate missing messages using third party services like Sentry
       escapeHtml: true, // disable escape html in variable mode
       commonLocaleDataUrls: COMMON_LOCALE_DATA_URLS,
-      fallbackLocale: null, // Locale to use if a key is not found in the current locale
+      fallbackLocale: '', /** @deprecated Locale to use if a key is not found in the current locale */
```

Comment:
I would suggest not to deprecate this option. It seems that your use case is very specific and this config would make break an otherwise stable feature. 

A suggestion: Your case could also be handled by using a comma separated fallbackLocale sequence like: 'zh-CN,en-US' which would require some extra effort for your case, but would keep the config simple for most users.